iot2000.coffee

```diff
@@ -0,0 +1,35 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	version: 1
+	slug: 'IOT2000'
+	aliases: [ 'IOT2020', 'IOT2040' ]
+	name: 'Siemens IOT2000'
+	arch: 'i386'
+	state: 'preview'
+
+	instructions: commonImg.instructions
+
+	gettingStartedLink:
+		windows: 'http://docs.resin.io/#/pages/installing/gettingStarted-IOT2000.md#windows'
+		osx: 'http://docs.resin.io/#/pages/installing/gettingStarted-IOT2000.md#on-mac-and-linux'
+		linux: 'http://docs.resin.io/#/pages/installing/gettingStarted-IOT2000.md#on-mac-and-linux'
```

Comment:
Please change the getting started links to something like: https://github.com/resin-os/resin-raspberrypi/blob/master/raspberrypi.coffee#L21

Coordinate with @zwalchuk about these links